org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/accessibility/AccessibleEditableTextEvent.d

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 2010, 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ * Port to the D programming language:
+ *     alice <stigma@disroot.org>
+ *******************************************************************************/
+module org.eclipse.swt.accessibility.AccessibleEditableTextEvent;
+
+import java.lang.all;
+import java.util.EventObject;
+
+/**
+ * Instances of this class are sent as a result of accessibility clients
+ * sending AccessibleEditableText messages to an accessible object.
+ *
+ * @see AccessibleEditableTextListener
+ * @see AccessibleEditableTextAdapter
+ *
+ * @since 3.7
+ */
+public class AccessibleEditableTextEvent : EventObject {
+
+    /**
+     * [in] 0-based start offset of the character range to perform
+     * the operation on
+     *
+     * @see AccessibleEditableTextListener#copyText
+     * @see AccessibleEditableTextListener#cutText
+     * @see AccessibleEditableTextListener#pasteText
+     * @see AccessibleEditableTextListener#replaceText
+     */
+    public int start;
+
+    /**
+     * [in] 0-based ending offset of the character range to perform
+     * the operation on
+     *
+     * @see AccessibleEditableTextListener#copyText
+     * @see AccessibleEditableTextListener#cutText
+     * @see AccessibleEditableTextListener#replaceText
+     */
+    public int end;
+
+    /**
+     * [in] a string that will replace the specified character range
+     *
+     * @see AccessibleEditableTextListener#replaceText
+     */
+    public String string_;
```

Comment:
Is this how it's done in other places in the code? `string` is not a keyword, just another symbol name.

Reply:
I couldn't (quickly) find any other occurrence of using `string` or `string_` as a field name. That said, there are times where `String string` is used (e.g. in [Label.setText]), so I'm happy to change it to `string` if that's preferred?

[Label.setText]: https://github.com/d-widget-toolkit/dwt/blob/202c52e/org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/widgets/Label.d#L571

Reply:
The preferred way is to aim for correctness first then the second goal is to be as close to the original Java source code (to make it easier to port new code in the future). It's go with `string`.

Reply:
That makes sense, I think you've said something similar before as well so I'll keep this in mind. I've updated the code.